src/BUILD

```diff
@@ -6,9 +6,7 @@ load(":rule_size_test.bzl", "rule_size_test")
 
 exports_files(["jdeps_modules.golden"])
 
-# We avoid using the `head` tool's `-c` option, since it does not exist on
-# OpenBSD; here we use `dd` instead.
-md5_cmd = "set -e -o pipefail && echo $(SRCS) | sort | xargs %s | %s | dd bs=32 count=1 > $@"
+md5_cmd = "set -e -o pipefail && echo $(SRCS) | xargs %s | %s | %s > $@"
```

Comment:
So now, can we do `%s $(SRCS)` and avoid the `echo` and `xargs` calls, or are these command lines too long for this to work?

Reply:
tests say we can dispense with `xargs`